XM_Dinner/362_Glalie.yaml

```diff
@@ -0,0 +1,22 @@
+species: Glalie
```

Comment:
I would cut the defensive investment here at least, it's already quite bulky and has an ohko move.